docs/initial-startup/07-macro-variables.md

```
@@ -22,7 +22,7 @@
 
 AFC has many variables that you can tweak for macros, including definition of complete custom macros to replace the
 stock ones. Most users will never need to touch these, and they will not be covered in this section. A full
-listing of all configuration options is available [here](../afc-klipper-add-on/configuration/configuration_overview.md). Below are the
+listing of all configuration options is available [here](../configuration/configuration_overview.md). Below are the
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)